src/nupic/algorithms/Connections.hpp

```diff
@@ -337,6 +347,13 @@ class Connections : public Serializable
    */
   size_t segmentFlatListLength() const { return segments_.size(); };
 
+  /**
+   * Get the vector length needed to use synpase indices.
+   *
+   * @retval A vector length
+   */
+  size_t synapseFlatListLength() const { return synapses_.size(); };
```

Comment:
is this used outside of Conn? I've seen it only internally, since it's a simple statement, please prefer `synapses_.size()`